src/components/Navigations/NavBar/index.js

```diff
@@ -1,14 +1,32 @@
-/* eslint-disable linebreak-style */
 /* eslint-disable no-unused-vars */
-import React from 'react';
+/* eslint-disable class-methods-use-this */
+import React, { Component, Fragment } from 'react';
+import Navbar from 'react-bootstrap/Navbar';
 
-import NavigationItem from '../NavigationItem';
-import classes from './style.css';
+class Index extends Component {
+  render() {
+    return (
+      <Fragment>
+        <Navbar bg="dark" variant="dark" expand="lg">
+          <Navbar.Brand href="#home">Broomy</Navbar.Brand>
+        </Navbar>
+        {/* authenticated = (
+        <Navbar bg="dark" variant="dark">
+        <Navbar.Brand href="#home">
+          <img alt="" src="/logo.svg" width="30" height="30" className="d-inline-block align-top" /> Broomy
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 107. Maximum allowed is 100.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5592511)